image/share/Taskfile

```diff
@@ -0,0 +1,3 @@
+default: KMER=121 PARTITIONS=4
+fast: KMER=63 PARTITIONS=128
+faster: KMER=31 PARTITIONS=256
```

Comment:
Regarding the tasks, for a user who is not familiar with velour how would they know the when they should use k31c4_01 or k31c20_01?

I think in the past we have tried to make the task bundles more transparent by giving them human readable names. Do you think you could translate the task bundles you previously had into something along those lines?